servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/gc/AbstractRestGCTest.java

```diff
@@ -25,7 +25,7 @@
 import org.projectnessie.gc.base.IdentifiedResult;
 import org.projectnessie.model.Branch;
 
-public abstract class AbstractRestGCTest extends AbstractRestGC {
+public abstract class AbstractRestGCTest extends AbstractRestGCRepoTest {
```

Comment:
This is just an extension to run all the test cases from single consumer class, similar to `AbstractRestTest` hierarchy.